config/class-site-details-index.php

```diff
@@ -228,6 +248,74 @@ public function get_jetpack_info() {
 		return $jetpack_info;
 	}
 
+	/**
+	 * Gather all the information about Parsely
+	 * @return array Parsely plugin info.
+	 */
+	public function get_parsely_info() {
+		$parsely      = get_option( 'parsely' );
+		$parsely_info = [];
+
+		$parsely_info['active']           = false;
+		$parsely_info['integration_type'] = $this->get_parsely_integration_type();
+
+		if ( self::PARSELY_INTEGRATION_TYPE_MUPLUGINS === $parsely_info['integration_type'] ||
+			self::PARSELY_INTEGRATION_TYPE_MUPLUGINS_SILENT === $parsely_info['integration_type'] ||
+			self::PARSELY_INTEGRATION_TYPE_SELF_MANAGED === $parsely_info['integration_type']
+		) {
+			$parsely_info['active'] = true;
+		}
+
+		$parsely_info['service_type'] = self::PARSELY_SERVICE_TYPE_NONE;
+
+		if ( self::PARSELY_INTEGRATION_TYPE_MUPLUGINS === $parsely_info['integration_type'] ||
+			self::PARSELY_INTEGRATION_TYPE_SELF_MANAGED === $parsely_info['integration_type']
+		) {
+			$parsely_info['service_type'] = self::PARSELY_SERVICE_TYPE_PAID;
+		}
+
+		$parsely_info['version'] = $this->get_parsely_version( $parsely );
+
+		return $parsely_info;
+	}
+
+	/**
+	 * Gets the Parsely version
+	 * @param array $parsely The Parsely options.
+	 * @return string The version.
+	 */
+	public function get_parsely_version( $parsely ) {
+		if ( ! $parsely || ! isset( $parsely['plugin_version'] ) ) {
+			return self::PARSELY_VERSION_NONE;
+		}
+
+		return $parsely['plugin_version'];
+	}
+
+	/**
+	 * The way that the plugin is used in WordPress
+	 * @return string integration type.
+	 */
+	public function get_parsely_integration_type() {
+		// detect a hidden installation
+		if ( false !== get_option( '_wpvip_parsely_mu', false ) && ! has_filter( 'wpvip_parsely_load_mu' ) ) {
```

Comment:
I think that `false !==` is not needed here?

Reply:
Thank you for the feedback. I have made this change in the most recent commit.